policy/release/cve.rego

```diff
@@ -141,17 +141,79 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [])
 }
 
+# METADATA
+# title: Deprecated task name
```

Comment:
"Deprecated result name"? Or maybe "Deprecated CVE result name" to stay in line with the short name.

Reply:
Went with the latter.